src/resources/js/commerce/gateway/stripe/checkout.js

```diff
@@ -500,16 +500,16 @@ tribe.tickets.commerce.gateway.stripe.checkout = {};
 	 * @link https://stripe.com/docs/js/elements_object/create_element?type=card#elements_create-options
 	 *
 	 * @since 5.3.0
+	 * @since TBD Pulled out `options` variable to allow filtering using `tec_tickets_commerce_stripe_checkout_localized_data`.
 	 */
 	obj.setupCompactCardElement = () => {
-		// Instantiate the CardElement with a single field combo.
-		obj.cardElement = obj.stripeElements.create( 'card', {
-			style: obj.checkout.cardElementStyle,
-		} );
+		// Instantiate the CardElement with the options
+		obj.cardElement = obj.stripeElements.create( 'card', options );
```

Comment:
where is `options` defined ?

Reply:
Good catch! I had it pulled out as a variable, which was unnecessary, but forgot to add the actual object back to the `options` arg... adding now!

Reply:
```suggestion
		obj.cardElement = obj.stripeElements.create( 'card', obj.checkout.cardElementOptions );
```

Reply:
I discussed this with Gustavo and he suggested keeping the previous `obj.checkout.cardElementStyle` as it was in case users had already been customizing it.